packages/mco/components/modals/app-manage-policies/helper/pvc-details-wizard-content.tsx

```diff
@@ -73,13 +74,14 @@ const getLabelsFromSearchResult = (searchResult: SearchResult): string[] => {
 
 const getLabelsFromTags = (tags: TagsType, currIndex: number): string[] =>
   tags.reduce((acc, tag, index) => {
-    const labels: string[] = (tag?.[1] || []) as string[];
+    const labels: string[] = (tag?.[NameValueEditorPair.Value] ||
+      []) as string[];
```

Comment:
nits: u r already doing `const labels: string[] =`
```suggestion
    const labels: string[] = tag?.[NameValueEditorPair.Value] ||
      [];
```